src/components/Card.module.css

```diff
@@ -0,0 +1,74 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  width: 343px;
+  min-height: 188px;
+  position: relative;
+  border-radius: 15px;
+  border: 0.5px #ebebec solid;
+  justify-content: space-between;
+  align-items: center;
+}
+.corner_wrapper {
+  display: flex;
+  flex-direction: row;
+  width: 343px;
+  height: 40px;
+  justify-content: flex-end;
+}
+.corner_button {
+  position: relative;
+  background: transparent;
+  border: none;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+.text_wrapper {
+  position: relative;
+}
+.question_text {
+  position: relative;
+  bottom: 0;
+  color: #161616;
```

Comment:
**nitpick:** It would be good to declare some CSS variables for tokens like color, font size, etc. so you will avoid duplicating these. It will also be easier to maintain